test/Microsoft.AspNetCore.Mvc.FunctionalTests/HttpResponseMessageExceptions.cs

```diff
@@ -7,7 +7,7 @@
 using Xunit;
 using Xunit.Sdk;
 
-namespace FiltersWebSite
+namespace Microsoft.AspNetCore.Mvc.FunctionalTests
```

Comment:
Do we actually want this thing to be public? Wouldn't you just try-catch the exception if you expect the server to throw?